app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -291,7 +291,9 @@ a.btn-info span{
   padding-bottom: 4px;
 }
 
-span.status-error, span.status-ok, span.status-warn, span.status-question {
-  padding-left: 3px;
+#properties_table {
```

Comment:
Please move this to hosts.scss, this has nothing to do with patternfly overrides.